packages/pass-style/doc/copyArray-guarantees.md

```diff
@@ -17,8 +17,7 @@ The input validation check `assertCopyArray(arr)` asserts that `passStyleOf(arr)
 # How do I enumerate thee, let me list the ways
 
 Why these properties restrictions?
-JavaScript has a [tremendous number of different constructs for enumerating the
-properties](enumerating-properties.md) of an object, with different semantics
+JavaScript has a [tremendous number of different constructs for enumerating the properties](enumerating-properties.md "https://github.com/endojs/endo/blob/master/packages/pass-style/doc/enumerating-properties.md") of an object, with different semantics
```

Comment:
the contents of the repo should not assume where the repo is located on the web or what branch it's in.

let's have a Slack convo to figure out an alternate solution to the problem

Reply:
> the contents of the repo should not assume where the repo is located on the web or what branch it's in

fair enough